i18n/en.json

```
@@ -53,8 +53,8 @@
     "processing": "Wait until processing is complete before leaving this page. This may take a few minutes.",
     "save": "Save renamed bundle/feed names",
     "select": "or select an existing one",
-    "notice": "You will need to wait a few minutes for new OSM and GTFS feeds to be processed.",
-    "createDescription": "When creating a new Network Bundle you have the option of reusing data from previously created Network Bundles. Note: Network Bundles created before April 15th, 2020 use OSM data uploaded to the region.",
+    "notice": "After clicking \"Create\" you will need to wait a few minutes for new OSM and GTFS feeds to be processed.",
+    "createDescription": "A Network Bundle requires both GTFS and OSM data. When creating a new Network Bundle you have the option of reusing data from previously created Network Bundles within this region. Note: Network Bundles created before April 15th, 2020 use OSM data uploaded to the region.",
```

```suggestion
    "createDescription": "A Network Bundle requires both GTFS and OSM data. When creating a new Network Bundle you have the option of reusing data from previously created Network Bundles within this region.",
```

I think we no longer need the note about OSM moving from region to bundle six months ago. If someone hasn't logged in since then, they probably don't remember the old setup anyways.